virttest/qemu_vm.py

```diff
@@ -1974,7 +1948,7 @@ def sort_key(dev):
 
                 # Handle the '-net nic' part
                 virtio = "virtio" in nic_model
-                parent_bus = _get_pci_bus(devices, nic_params, "nic", virtio)
+                parent_bus = self._get_pci_bus(nic_params, "nic", virtio)
```

Comment:
@xutian  `e1000e` is a pcie device, but with `virtio = "virtio" in nic_model` is `False`,  which case `pcie=False`  in `_get_pci_bus`.  this device will be added to `pcie-pci-bridge-0`.

Reply:
@xutian  It is an existing bug which is not related with this patch, we can fix with a separate commit:)

Reply:
fixed

Reply:
@xutian Thanks, I am ok with the fix, but a new commit is more better, it is not related with hotplug, how do you think?